docs/commands.rst

```diff
@@ -544,6 +544,35 @@ remote test machine.  You can run jobs on the local machine but it is not
 recommended since some tasks reboot the system. Hosts are tied to recipe IDs
 inside the job XML.
 
+Arguments for the client are as follows::
+
+    restraint --host <1=host> --job <job.xml> [--restraint-path </dir/restraintd>] [-v]
+
+Where:
+
+.. option:: --host <recipe_id>=[<user>@]<host>
+
+   Set host for a recipe with specific id.
+
+.. option:: --job </yourdir/your-job.xml>
+
+   File Location of your job.xml.
+
+.. option:: --restraint-path </dir/restraintd>
+
+   The optional argument ``--restraint-path`` passes the directory from which to run
+   `restraintd` from.  This can be used by developers where the
+   restraint repo is pulled and ``restraintd`` image is built.  By default, the
+   installed image is executed.
```

Comment:
This option is for the path of the restraintd binary to run, binary included. It's not only for the directory. Adding to what was written in the option help,

> specify the path to the restraintd binary to run on the remote machine

Reply:
done